Terminology Update Blog Post (#3467)
jenkins-infra/jenkins.io: content/blog/2020/06/2020-06-18-terminology-update.adoc

---
layout: post
title: "Terminology Update"
tags:
- community

> - opensource

Reviewer: ```suggestion
- opensource
- governance
- documentation
```

author: markyjackson-taulia
description: "Terminology Updates"
opengraph:

>   image: /images/docker/dockerJenkins_social.png

Reviewer: I am not sure we want to use the Docker image here

---

> In 2016, the Jenkins community decided to start removing offensive terminology within the project. Offensive language was deprecated in Jenkins 2.0.

Reviewer: ```suggestion
In 2016, the Jenkins community decided to start removing offensive terminology within the project.
The "slave" term was deprecated in Jenkins 2.0 and replaced by the "agent" term.
Other terminology was slated for review after the cleanup of the "slave" term which was considered as most problematic one.
```

In 2017, the project began tracking areas for correction link:https://issues.jenkins-ci.org/browse/JENKINS-42816[tracking areas for correction].

> This year, a group of core contributors continued addressing this critical link:https://www.jenkins.io/blog/2020/05/06/docker-agent-image-renaming/[work]. 

Reviewer: Might make sense to mention the SSH Build Agents plugin renaming and gradual cleanup of the terminology in services and repositories

> The Advocacy & Outreach SIG met to discussed and prioritize the continued work. The goverence board has also met and there will be more information coming regarding removal of offsenive terminology.

Reviewer: i think we need to add meeting minuts and recordings for both meetings. Oh well, recordings... Will do them today

> Last week we took another step towards removing offensive terminology within the project by updating previous blog posts and link:https://github.com/jenkins-infra/jenkins.io/pull/3447[removing offensive terminology].

Reviewer: ```suggestion
Last week we took another step towards removing offensive terminology within the project by updating previous blog posts and link:https://github.com/jenkins-infra/jenkins.io/pull/3447[removing offensive terminology in old blogs], cleaning up some references in Jenkins built-in documetation and localization, etc.
```

There were multiple pull request over the

> There is more link:https://groups.google.com/forum/#!topic/jenkinsci-dev/CLR55wMZwZ8[work to do].

Reviewer: Would make sense to explicitly list the terms ("master", "whitelist", "blacklist") we want to remove, and our action plan for that. Hinting the cleanup of the Git barnch terminology could also make sense in this context

We continue to do this much needed work and would like to remind everyone that the Jenkins project is governed by the link:https://www.jenkins.io/project/conduct/[Code of Conduct.]

> 

Reviewer: I think it makes sense to add a Call for Action for "How to contribute?". I can help with this section

Author: I disagree. This blog post is to discuss the work that has already been done and noting we are still working. If this blog post makes people become passionate, they will ask to help.

---
layout: post
title: "On Jenkins Terminology Updates"
tags:
- community
- opensource

> - goverence

Reviewer 2: ```suggestion
- governance
```

- documentation
author: markyjackson-taulia
description: "Terminology Updates"

> opengraph:

Reviewer 2: If not providing an opengraph image, probably  better to not have the empty tag
```suggestion
```

---

In 2016, the Jenkins community decided to start removing offensive terminology within the project.
The "slave" term was deprecated in Jenkins 2.0 and replaced by the "agent" term.
Other terminology was slated for review after the cleanup of the "slave" term which was considered as most problematic one.
In 2017, the project began link:https://issues.jenkins-ci.org/browse/JENKINS-42816[tracking areas for correction].
Work has been done on renaming the SSH build agent plugin as well as gradual removal of offensive naming in services and repositories.
This year, a group of core contributors continued addressing this critical link:https://www.jenkins.io/blog/2020/05/06/docker-agent-image-renaming/[work].

> The Advocacy & Outreach SIG met to discuss and prioritize the continued work. The goverence board has also met and there will be more information coming regarding removal of offsenive terminology.

Reviewer 2: Spelling fixes
```suggestion
The Advocacy & Outreach SIG met to discuss and prioritize the continued work. The governance board has also met and there will be more information coming regarding removal of offensive terminology.
```

> The Advocacy & Outreach SIG met to discussed and prioritize the continued work. The goverence board has also met and there will be more information coming regarding removal of offsenive terminology.

Reviewer 3: remove last 'd' on discussed

Author: fixed @tracymiranda

Last week we took another step towards removing offensive terminology within the project by updating previous blog posts and link:https://github.com/jenkins-infra/jenkins.io/pull/3447[removing offensive terminology in old blogs], cleaning up some references in Jenkins built-in documetation and localization, etc.
The meeting minutes are available link:https://docs.google.com/document/d/1K5dTSqe56chFhDSGNfg_MCy-LmseUs_S3ys_tg60sTs/edit[here] and a recording of the meeting link:https://www.youtube.com/watch?v=hGZ1CvkmZXU[here]
There is more link:https://groups.google.com/forum/#!topic/jenkinsci-dev/CLR55wMZwZ8[work to do]. The core team is working to address terms such as "Master", "whitelist" and "blacklist" as well addressing git branching terminology.

We could use link:https://www.jenkins.io/participate/[your help]
We continue to do this much needed work and would like to remind everyone that the Jenkins project is governed by the link:https://www.jenkins.io/project/conduct/[Code of Conduct.]

Sincerely,

> Marky Jackson

Reviewer 3: LGTM
